docs/api/rest_api/rest_api_reference/input/ibexa-types.raml

```diff
@@ -2195,10 +2239,71 @@ Segment:
     description: 'This class represents a segment.'
     type: BaseObject
     properties:
-        id: integer
-        identifier: string
-        name: string
-        SegmentGroup: SegmentGroup
+        id:
+            description: 'A unique numeric identifier for the segment.'
+            type: integer
+        identifier:
+            description: 'A unique numeric identifier for the segment.'
+            type: string
+        name:
+            description: 'A name of the segment.'
+            type: string
+        SegmentGroup:
+            description: 'A reference to the segment group to which this segment belongs.'
+            type: object
+            properties:
+                media-type:
+                    description: 'Mmedia type of the segment group resource.'
+                    type: string
+                href:
+                    description: 'A URL reference to the segment group resource.'
+                    type: string
+
+SegmentCreate:
+    description: 'This class represents a new segment to create.'
+    type: object
+    properties:
+        identifier:
+            description: 'A unique identifier for the segment.'
+            type: string
+            required: true
+        name:
+            description: 'A name of the segment.'
+            type: string
+            required: true
+        group:
+            description: 'A segment group to which the segment belongs.'
+            type: string
+            required: true
+
+SegmentCreateWrapper:
+    description: 'JSON object with only a SegmentCreate property.'
+    type: object
+    properties:
+        SegmentCreate: SegmentCreate
+
+SegmentUpdate:
+    description: 'This class represents an update to an existing segment.'
+    type: object
+    properties:
+        identifier:
+            description: 'A unique identifier for the segment.'
+            type: string
+            required: true
+        name:
+            description: 'A name of the segment.'
+            type: string
+            required: true
+        group:
+            description: 'A group to which the segment belongs.'
```

Comment:
```suggestion
            description: 'Segment group identifier to which the segment belongs.'
```
It's important to always specify what kind of value is expected (ID or identifier). It's in the examples as well, but it makes the reference easier to use.